mdx2/command_line/scale.py

```
@@ -2,328 +2,298 @@
+    ny: int = 20  # number of grid control points in the y direction
+    dphi: float = 5.0  # spacing of phi control points in degrees
+    niter: int = 5  # maximum iterations in refinement
+    x2tol: float = 1.0e-4  # maximum change in x2 to stop refinement early
+    outlier: float = 5.0  # standard error cutoff for outlier rejection after refinement
+
+
+@dataclass
+class Parameters:
+    """Options for refining a scaling model to unmerged corrected intensities"""
+
+    hkl: List[str] = field(positional=True, nargs="+")  # NeXus file(s) containing hkl_table
```

Ruff (0.14.8)

Do not perform function call `field` in dataclass defaults

(RUF009)